### PR TITLE
HardwareView: Get GPU info asynchronously

### PR DESCRIPTION
**Pull request by davidmhewitt**
Don't block the constructor (and hence the plug loading) by using synchronous DBus calls.

### CODE REVIEW

File: src/Views/HardwareView.vala [hunk 1]
```diff
@@ -289,6 +298,18 @@ public class About.HardwareView : Gtk.Grid {
...
     }
 
+    private async void get_graphics_info () {
+        var primary_gpu = yield get_gpu_info (true);
+        primary_graphics_info.label = primary_gpu;
+
+        var secondary_gpu = yield get_gpu_info (false);
+        if (secondary_gpu != null) {
+            secondary_graphics_info.label = secondary_gpu;
+            graphics_grid.add (secondary_graphics_info);
```
Comment (danrabbit) [inline src/Views/HardwareView.vala:308]: Another way to do this without adding an additional grid would be to use https://valadoc.org/gtk+-3.0/Gtk.Grid.attach_next_to.html

**Review by danrabbit — CHANGES_REQUESTED** [review]
I think we can save a grid with `attach_next_to` :)

File: src/Views/HardwareView.vala (hunk 1 above)
Comment (davidmhewitt) [reply src/Views/HardwareView.vala:308]: Hm, I tried that, but it doesn't seem to update the items below it in the grid, so you just end up with two labels on top of each other.

Comment (davidmhewitt) [reply src/Views/HardwareView.vala:308]: @danrabbit Here's what I get. The second GPU label correctly attaches below the first, but doesn't push down the subsequent RAM and storage labels:

Screenshot from 2021-01-27 17-29-05

Comment (danrabbit) [reply src/Views/HardwareView.vala:308]: Oh you're right. That's lame. Nevermind then!

**Review by danrabbit — APPROVED** [review]

**Pull request merged**